packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,55 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  - The token rates controller `chainId` setter now expects the chain ID as `Hex` rather than as a decimal string
+  - The tokens controller now expects the `chainId` configuration entry and constructor parameter as type `Hex` rather than decimal `string`
+  - The tokens controller `addDetectedTokens` method now accepts the `chainId` property of the `detectionDetails` parameter to be of type `Hex` rather than decimal `string`.
+  - The tokens controller state properties `allTokens`, `allIgnoredTokens`, and `allDetectedTokens` are now keyed by chain ID in `Hex` format rather than decimal `string`.
+    - This requires a state migration
+- **BREAKING**: Use approval controller for suggested assets ([#1261](https://github.com/MetaMask/core/pull/1261), [#1268](https://github.com/MetaMask/core/pull/1268))
+  - The actions `ApprovalController:acceptRequest` and `ApprovalController:rejectRequest` are no longer required by the token controller messenger.
+  - The `suggestedAssets` state has been removed, which means that suggested assets are no longer persisted in state
+  - The return type for `watchAsset` has changed. It now returns a Promise that settles after the request has been confirmed or rejected.
+- **BREAKING:** Initialize controllers with the current network ([#1361](https://github.com/MetaMask/core/pull/1361))
+  - The following controllers now have a new `chainId` required constructor parameter:
+    * `AssetsContractController`
+    * `NftController`
+    * `NftDetectionController`
+    * `TokenRatesController`
+    * `TokensController`
+- **BREAKING:** The token list controller messenger requires the `NetworkController:stateChange` event instead of the `NetworkController:providerConfigChange` event ([#1329](https://github.com/MetaMask/core/pull/1329))
+- **BREAKING:** The token list controller `onNetworkStateChange` option now has a more restrictive type ([#1329](https://github.com/MetaMask/core/pull/1329))
+  - The event handler parameter type has been changed from `NetworkState | ProviderConfig` to `NetworkState`
+- **BREAKING:** Update the account tracker controller `provider` type ([#1266](https://github.com/MetaMask/core/pull/1266))
+  - The `provider` setter and the `provider` config entry now use our `Provider` type from `eth-query` rather than `any`
+- Bump @metamask/abi-utils from 1.1.0 to 1.2.0 ([#1287](https://github.com/MetaMask/core/pull/1287))
+- Bump @metamask/utils from 5.0.1 to 5.0.2 ([#1271](https://github.com/MetaMask/core/pull/1271))
+
+### Removed
+- **BREAKING:** Remove the `networkType` configuration option from the NFT detection controller, NFT controller, and tokens controller ([#1360](https://github.com/MetaMask/core/pull/1360), [#1359](https://github.com/MetaMask/core/pull/1359))
+- **BREAKING:** Remove the `SuggestedAssetMeta` type from the token controller ([#1268](https://github.com/MetaMask/core/pull/1268))
```

Comment:
Are we missing these changes also introduced by #1268?

- The `SuggestedAssetMetaBase` type was removed
- The `acceptWatchAsset` and `rejectWatchAsset` methods were removed from TokensController

Reply:
Good catch - more detail added here: https://github.com/MetaMask/core/pull/1385/commits/1ca5bc168298d6bdd0fabd472a99af8be8c7b5a7